test/derivative_operators_interface.jl

```diff
@@ -12,28 +12,34 @@ end
 
 # Analytic solutions to higher order operators.
 # Do not modify unless you are completely certain of the changes.
+
+# 4th derivative, 4th order
 function fourth_deriv_approx_stencil(N)
     A = zeros(N,N+2)
     A[1,1:8] = [3.5 -56/3 42.5 -54.0 251/6 -20.0 5.5 -2/3]
     A[2,1:8] = [2/3 -11/6 0.0 31/6 -22/3 4.5 -4/3 1/6]
-    A[N-1,N-5:end] = reverse([2/3 -11/6 0.0 31/6 -22/3 4.5 -4/3 1/6], dims=2)
-    A[N,N-5:end] = reverse([3.5 -56/3 42.5 -54.0 251/6 -20.0 5.5 -2/3], dims=2)
+
+    A[N-1,N-5:end] = reverse([3.5 -56/3 42.5 -54.0 251/6 -20.0 5.5 -2/3], dims=2)
```

Comment:
@ajozefiak double check this change.

Reply:
Yes its the old stencil only. The order was opposite as I was reading them opposite in the convolutions.jl file.